release/models/bfd/openconfig-bfd.yang

```diff
@@ -26,7 +26,14 @@ module openconfig-bfd {
     "An OpenConfig model of Bi-Directional Forwarding Detection (BFD)
     configuration and operational state.";
 
-  oc-ext:openconfig-version "0.3.0";
+  oc-ext:openconfig-version "0.3.1";
```

Comment:
Since we're adding a feature here, let's make this be a minor release - i.e., this should be `0.4.0`.

Reply:
@Pull-eckermann please update version to 0.4.0.  Thanks!